include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2556,6 +2556,21 @@ class FEEvaluation : public FEEvaluationAccess<dim,
     const unsigned int active_fe_index   = numbers::invalid_unsigned_int,
     const unsigned int active_quad_index = numbers::invalid_unsigned_int);
 
+  /**
+   * Constructor. Takes all data stored in MatrixFree for a given cell range.
```

Comment:
Can you move the last part of the comment, i.e., lines 2565-2566, here to make clear what the cell range refers to: I would say "... for a given cell range, which allows to automatically identify the active_fe_index and active_quad_index in case of a p-adaptive strategy".

Reply:
OK. I have changed this!